how_and_what_to_document.md

```diff
@@ -0,0 +1,32 @@
+# GitBook & GitHub Workflow
+
+The following chapter assumes you are working with GitBook UI. As a developer you may feel comfortable working with your text editor and command line. Both ways are fine, as long as the end results will be a GitHub pull request.
+
+GitBook's repository is being automatically synced to GitHub, so you may change it in one of them, and it is guaranteed to be reflected in the other.
+
+As a privileged user, start by [editing](https://www.gitbook.com/book/amitaibu/the-gizra-way/details) the book
+
+
+
+![](images/gitbook/image1.jpg)
+
+Create a new branch, you will work on a side branch, which will not immediately be reflected on the official book. This allows you to go wild with your edits, without the fear of breaking the live site.
```

Comment:
Not sure I edited this correctly about the new and side brance (which are the same).